ssvsigner/cmd/ssv-signer/ssv-signer.go

```diff
@@ -1,53 +1,48 @@
 package main
 
 import (
-	"errors"
 	"fmt"
 	"log"
 	"net/url"
 	"time"
 
 	"github.com/alecthomas/kong"
 	"github.com/herumi/bls-eth-go-binary/bls"
-	"github.com/valyala/fasthttp"
 	"go.uber.org/zap"
 
 	"github.com/ssvlabs/ssv/ssvsigner"
 	"github.com/ssvlabs/ssv/ssvsigner/keys"
 	"github.com/ssvlabs/ssv/ssvsigner/keystore"
+	"github.com/ssvlabs/ssv/ssvsigner/tls"
 	"github.com/ssvlabs/ssv/ssvsigner/web3signer"
 )
 
 type CLI struct {
 	ListenAddr         string        `env:"LISTEN_ADDR" default:":8080" required:"" help:"The address and port to listen on (e.g. :8080)"` // TODO: finalize port
-	Web3SignerEndpoint string        `env:"WEB3SIGNER_ENDPOINT" required:"" help:"URL of the web3signer service"`
+	Web3SignerEndpoint string        `env:"WEB3SIGNER_ENDPOINT" required:"" help:"URL of the web3signer service" name:"web3signer-endpoint"`
```

Comment:
@nkryuchkov can you confirm `name` parameter? I added cli tests and without the parameter I had: `web-3-signer-ednpoint` flag

Reply:
@kchojn it looks good to me